### PR TITLE
Api student controller minor change

### PR DESCRIPTION
**Pull request by JazibEqbal**
app.js accidently stashed but re-fixed to original.

### CODE REVIEW

File: src/model/student.js [hunk 1]
```diff
@@ -28,13 +28,13 @@ const studentSchema = new mongoose.Schema({
   },
   password:{
       type: String,
-      required: [false, "Please provide password"],
+      required: [true, "Please provide password"],
```
Comment (imfAli) [inline src/model/student.js]: This will be false now, since Google OAuth wont consist password entry.

File: src/model/student.js [hunk 2]
```diff
@@ -28,13 +28,13 @@ const studentSchema = new mongoose.Schema({
...
       minLength: 8,
       trim: true,
   },
   phone: {
       type: Number,
-      required: [false, "Please provide your phone number"],
+      required: [true, "Please provide your phone number"],
```
Comment (imfAli) [inline src/model/student.js]: Same as above

File: src/api/controller/student.controller.js [hunk 3]
```diff
@@ -105,15 +103,15 @@ exports.createOrder = async (req, res) => {
...
     else {
       saveOrder = await Orders.updateSchema(req.body.orderid, req.body);
     }
     res.status(StatusCodes.OK).send({
       status: 'Order made successfully',
       data: {
         order: saveOrder
-      }
+      },
+      isNewMember: saveOrder.isNewMember
```
Comment (imfAli) [inline src/api/controller/student.controller.js]: Why is this needed?

Comment (JazibEqbal) [reply src/api/controller/student.controller.js]: Resolved

File: src/model/student.js (hunk 2 above)
Comment (JazibEqbal) [reply src/model/student.js]: Resolved

File: src/model/student.js (hunk 1 above)
Comment (JazibEqbal) [reply src/model/student.js]: Resolved
